proccontrol/src/procset.C

```diff
@@ -45,10 +45,6 @@
 #include <map>
 #include <algorithm>
 
-#if defined(os_windows)
```

Comment:
Did we need to include stdint.h and inttype.h here, or are they not used in this file?

Reply:
They are not necessary for this file.